code/go/internal/validator/spec_test.go

```diff
@@ -5,9 +5,10 @@
 package validator
 
 import (
-	"github.com/elastic/package-spec/code/go/internal/fspath"
 	"testing"
 
+	"github.com/elastic/package-spec/code/go/internal/fspath"
+
```

Comment:
Nit. Move package spec imports to the bottom of the import block.

Reply:
I thought goimports would reorder it and I didn't check that. Updated.
Thanks!